mwe.py

```diff
@@ -3,12 +3,15 @@
 """
 
 import asyncio
+from pathlib import Path
 
-from edi_energy_scraper import EdiEnergyScraper
+from edi_energy_scraper.scraper import EdiEnergyScraper
+
+my_target_dir = Path(__file__).parent / "foo"
```

Comment:
```suggestion
my_target_dir = Path(__file__).parent / "foo"

my_target_dir.mkdir(exist_ok=True)
```

ist mir lokal auf die Füße gefallen

Reply:
dann hätte es geklappt